build-tools/packages/build-tools/src/typeValidator/packageJson.ts

```diff
@@ -49,13 +54,22 @@ function safeParse(json: string, error: string) {
 
 export async function getPackageDetails(packageDir: string): Promise<PackageDetails> {
 	const packagePath = `${packageDir}/package.json`;
-	if (!(await util.promisify(fs.exists)(packagePath))) {
+	if (!existsSync(packagePath)) {
 		throw new Error(`Package json does not exist: ${packagePath}`);
 	}
-	const content = await util.promisify(fs.readFile)(packagePath);
-
-	const pkgJson: PackageJson = safeParse(content.toString(), packagePath);
 
+	// try loading from typeValidation.config.json, then fall back to package.json
+	const configExplorer = cosmiconfigSync("typeValidation", {
+		searchPlaces: ["typeValidation.config.json", "package.json"],
```

Comment:
Nit: these file name strings are used in a few places. Should we create variables for them?

Reply:
Done.